### DISCUSSION TITLE
User Manual - EV Set Configuration Example

### DISCUSSION THREAD

**Discussion by Bobbybigears** [Sevenstax/FreeV2G#114] [General]
Hi,
In version 2.14 of the User Manual under section 18.9.1, there's an example of how to use the EV Set Configuration Command. Can you confirm that the example is correct? The payload size bytes seem to be incorrect, as does the way the Battery Capacity field is populated. The field is described as 'Exponential', but the value is literal, i.e. 0xC350 = 50000.  Apologies if I'm missing something obvious.
Best regards
Rob

**Comment by jpo-stx** [comment]
The example is wrong. Example for Set configuration should be:
```
C0 27 A0 00 00 10       (ReqID: 0; Payload: 16 byte)
01 02 03 04 05 06       (EV ID)
02 00 01                (Protocols DIN70121 and ISO15118 supported)
01 00                   (Payment method EIM)
01 01                   (Energy transfer mode DC extended)
00 32 03                (Battery capacity of 50,000Wh)
00 C1
```

**Comment by Bobbybigears** [comment]
Thank-you for the clarification. Much appreciated. Rob.
